### PR TITLE
Add org.gnome.gitlab.cheywood.Iotas

### PR DESCRIPTION
**Pull request by cheywood**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. Link:  [*gitlab.gnome.org/cheywood/Iotas*](https://gitlab.gnome.org/cheywood/Iotas)
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build org.gnome.gitlab.cheywood.Iotas

File: org.gnome.gitlab.cheywood.Iotas.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        "/lib/pkgconfig",
+        "/man",
+        "/share/aclocal",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig"
+    ],
+    "modules": [
+        {
+            "name" : "gtksourceview",
```
Comment (hfiguiere) [inline org.gnome.gitlab.cheywood.Iotas.json:28]: this version of gtksourceview is in the Sdk.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.gnome.gitlab.cheywood.Iotas.json (hunk 1 above)
Comment (cheywood) [reply org.gnome.gitlab.cheywood.Iotas.json]: Of course it is! Oops. :facepalm: Thanks.

**Comment by cheywood** [comment]
Looks good :+1:  Thanks again.

**Pull request closed without merging**